config/crd/bases/vmoperator.vmware.com_virtualmachines.yaml

```diff
@@ -1449,7 +1449,7 @@ spec:
                             is CloudInit. Please note it is up to the user to ensure
                             the provided device name does not conflict with any other
                             devices inside the guest, ex. dvd, cdrom, sda, etc."
-                          pattern: ^\w\w+$
+                          pattern: ^[a-z0-9][a-z0-9]+$
```

Comment:
Isn't the second character class missing `-`?

Reply:
I didn't think we needed to allow `-` for network names.  The earlier `\w` doesn't include `-` too, right?.

Reply:
We've discussed this offline and decided to go without allowing `-`. We will make it more lenient if required later.